prow/gerrit/client/client.go

```diff
@@ -602,7 +602,7 @@ func (h *gerritInstanceHandler) QueryChangesForProject(log logrus.FieldLogger, p
 
 	var opt gerrit.QueryChangeOptions
 	opt.Query = append(opt.Query, strings.Join(append(additionalFilters, "project:"+project), "+"))
-	opt.AdditionalFields = []string{"CURRENT_REVISION", "CURRENT_COMMIT", "CURRENT_FILES", "MESSAGES"}
+	opt.AdditionalFields = []string{"CURRENT_REVISION", "CURRENT_COMMIT", "CURRENT_FILES", "MESSAGES", "LABELS"}
```

Comment:
How was Tide considering merge requirements at all without this? We're labels looked up separately later? Why is this needed here now?

Reply:
it was unfortunately overlooked, the "required" functionality for Gerrit probably never worked :(

Reply:
Is there some integration test case that needs to be added here?

Reply:
integration tests won't be very helpful imo, unless we can hook up into a real gerrit server

Reply:
> , the "required" functionality for Gerrit probably never worked :(

We didn't see Tide immediately merging all changes though. It was retesting required jobs, right?

Reply:
I think right now it retest failed jobs that are still in api server